### PR TITLE
New package: tt-rss-19.8

### PR DESCRIPTION
**Pull request by non-Jedi**
I've never packaged anything php-adjacent before, and this whole thing seems rather fiddly. This is my best shot at doing so in a reasonable manner; let me know if it's still super bone-headed.

### CODE REVIEW

File: srcpkgs/tt-rss/INSTALL [hunk 1]
```diff
@@ -0,0 +1,5 @@
+case ${ACTION} in
+post)
+	chown -R _ttrss:_ttrss var/lib/tt-rss
```
Comment (ericonr) [inline srcpkgs/tt-rss/INSTALL:3]: I think you want to use `make_dirs` in the template.

File: srcpkgs/tt-rss/files/tt-rss-update/run [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+chown -R _ttrss:_ttrss /var/lib/tt-rss/
+chpst -u _ttrss:_ttrss php /usr/share/webapps/tt-rss/update.php --daemon
```
Comment (ericonr) [inline srcpkgs/tt-rss/files/tt-rss-update/run:4]: Services should exec into something

**Comment by Chocimier** [comment]
Usually database isn't listed in depends to support setup with db running on different machine, please remove.
Remove archs=noarch, it is deprecated.

**Comment by non-Jedi** [comment]
When I packaged this, I didn't realize that miniflux was already packaged and did functionally the same thing. Given the existence of miniflux, I have no desire to maintain this. I'll make sure not to delete the branch in case someone wants to pick up where I left off.

**Pull request closed without merging**